ajax/libs/d3-brush/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "d3-brush",
+  "filename": "d3-brush.min.js",
+  "description": "Select a one- or two-dimensional region using the mouse or touch.",
+  "keywords": [
+    "d3",
+    "d3-module",
+    "brush",
+    "interaction"
+  ],
+  "homepage": "https://d3js.org/d3-brush/",
+  "license": "BSD-3-Clause",
+  "author": {
+    "name": "Mike Bostock",
+    "url": "https://bost.ocks.org/mike"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/d3/d3-brush.git"
+  },
+  "npmName": "d3-brush",
+  "npmFileMap": [
+    {
+      "basePath": "build",
+      "files": [
+        "**/*"
```

Comment:
It'll be better if `package.js` can be excluded here, it appears in many old versions.